### ISSUE TITLE
Changing temp_directory in config file

### ISSUE THREAD

**Issue by CassandraHjo** [CAMI-challenge/CAMISIM#173]
I am looking to run multiple simulations at once on a cluster, and therefore want to have different temp directories. I have tried the following two methods: 

- Change the name of the temp_directory in the config.ini file to something else (for instance from `/tmp` (like in the default) to `/tmp2`)
- Change the name of the temp_directory by giving the full path to the "new" temp_directory

When doing this I get this error: `ERROR: [Validator 18394365136] Directory does not exist: '/tmp2'`
I therefore tried to make the temp directory inside the CAMISIM folder before starting the simulation, but I still get the same error message.

I am unsure if using the same temp_directory for two simulations (same input genomes, metadata and number of samples) running at once will cause any problems, but if so: how do I change the temp_directory in the config file?

**Comment by AlphaSquad** [comment]
Hi,
in contrast to the `out` directory, CAMISIM will create individual folders under the `/tmp` directory, so it should not be a problem to have multiple CAMISIM runs with the same `/tmp` directory. 
That is also the reason why changing the directory to `/tmp2` does not work - CAMISIM does not create the `temp_directory` directory but only folders "below" that. 
Also note that since the path starts with the `/` it is an absolute path/in the root path of your file system, so you probably are unable to create a new `/tmp2` directory without haveing root access. If you want the `temp_directory` to be within the CAMISIM path (which I would not necessarily recommend), you would have to choose `tmp` (without the leading `/`) as a relative path

**Comment by TakacsBertalan** [comment]
Hi! I am experiencing a similar error. Could you tell me how you managed to solve it?
My root /tmp directory doesn't have enough space for CAMISIM, so I specified an existing directory on an external hard drive with more than enough space, with it's absolute path. Still, I keep getting the following message:
"ERROR: [NcbiTaxonomy] Directory does not exist: '/tmp/tmp_36zttyq/delnodes.dmp'"

It seems like, even though I specified a new tmp, CAMISIM still tries to use the old one. I tried to specify the path as a relative path, but it gave the same result.
I would appreciate any help

**Comment by CassandraHjo** [comment]
I solved my problem by defining the full path to the temp directory. In my case I also needed to create the directory before I ran the pipeline as "CAMISIM does not create the temp_directory directory but only folders "below" that."

**Comment by TakacsBertalan** [comment]
Thank you for the quick answer!
Interesting, I think I am doing the same, I still get the error. I will open a new issue.
Thanks for the help!
